### ISSUE TITLE
Maintaining connection with Github repository changing owners

### ISSUE THREAD

**Issue by noamross** [zenodo/zenodo#1653]
I have a GitHub repo linked to Zenodo that has already had a couple of Zenodo releases.  I want to transfer ownership of the GitHub repository to another GitHub user.  Will Zenodo still uptake releases from that other repository? Can we switch the "watched" repository of a Zenodo record or direct a new repository to an existing record?  Thanks!

**Comment by feschmidt** [comment]
Are there any plans to move further on this? I have a repository that had one release, then ownership was transferred and I now tried to make a second release, via github itself which worked fine.
However, it seems that while Zenodo recognizes a change of ownership on the user's github page (`https://zenodo.org/account/settings/github/`), clicking on the repository release page under `https://zenodo.org/account/settings/github/repository/<new username>/<old repo name>`, the repo and releases seem to still be linked to the old user, and the new releases return 
`
{
    "errors": "Unknown error occured."
}
`

**Comment by feschmidt** [comment]
and indeed the link to `Create release...` on the Zenodo github page still links to `https://github.com/<old user>/<repo>/releases/new` instead of `https://github.com/<new user>/<repo>/releases/new`

**Comment by MuellerSeb** [comment]
anything new on this? I am also planning to transfer a GitHub repo to another organization and don't want the Zenodo releases to break.

**Comment by sappelhoff** [comment]
I am also interested in this feature.

**Comment by samapriya** [comment]
Yeah stuck in the same situation here, I can create a release but since Zenodo cannot find the GitHub repo on my account anymore the release doesn't seem to be picked up by Zenodo

**Comment by RoyMudie** [comment]
@benbovy - don't suppose you've tried making a new release since the change to make sure this actually works now?

Facing the same problem. Leaving my current job and wanting to transfer ownership to a generic deparmental account. Obviously don't want to leave it broken :)

**Comment by benbovy** [comment]
> @benbovy - don't suppose you've tried making a new release since the change to make sure this actually works now?

Not yet unfortunately, but I'll try to report here when we'll do so.

**Comment by jsdodge** [comment]
The GitHub [documentation](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#whats-transferred-with-a-repository) says that it automatically redirects the original URL to the transfer destination URL. This worked for me. I also found that I could update the URL in the Zenodo metadata without creating a new version.

**Comment by jcolomb** [comment]
Anyone would like to test this workaround  https://book.the-turing-way.org/communication/citable/citable-cite.html?highlight=gitlab%20zenodo

the tool was meant for gitlab, but should be modifiable to work with github (?) https://gitlab.com/sbeniamine/gitlab2zenodo
